### PR TITLE
Bug 1209627 remove cruft for initial landing

### PR DESCRIPTION
**Pull request by oyiptong**
Remove things we won't need for the initial landing

r? @marcoscaceres, @emtwo or @sarracini ?

### CODE REVIEW

File: browser/components/newtab/RemoteNewTabLocation.jsm [hunk 1]
```diff
@@ -7,7 +7,8 @@ this.EXPORTED_SYMBOLS = ["RemoteNewTabLocation"];
 Components.utils.import("resource://gre/modules/Services.jsm");
 Components.utils.importGlobalProperties(["URL"]);
 
-const DEFAULT_PAGE_LOCATION = "https://mozilla.github.io/remote-newtab/";
+// TODO: will get dynamically set in bug 1210478
+const DEFAULT_PAGE_LOCATION = "https://newtab.cdn.mozilla.net/v0/nightly/en-US/index.html";
```
Comment (marcoscaceres) [inline browser/components/newtab/RemoteNewTabLocation.jsm:11]: :+1:

File: browser/components/newtab/RemoteAboutNewTab.jsm [hunk 2]
```diff
@@ -107,6 +71,16 @@ let RemoteAboutNewTab = {
   },
 
   /**
+   * Inits the content iframe with the newtab location
+   */
+  initContentFrame: function(message) {
```
Comment (marcoscaceres) [inline browser/components/newtab/RemoteAboutNewTab.jsm:74]: :+1:

**Comment by marcoscaceres** [comment]
looks good to me. However, please everyone check the messages that were deleted.

**Comment by oyiptong** [comment]
closes #22

**Comment by emtwo** [comment]
r+ looks good to me too

**Pull request closed without merging**